docs/getting-started.md

```diff
@@ -71,6 +71,10 @@ If you want to test it locally, you can skip this step.
 genezio login
 ```
 
+:::info
+If you are using Windows, then you need `node` version 18 or higher to be able to run this command properly.
```

Comment:
this should be 2 Infos
1. To install the genezio CLI you must have Node.JS >= 18.2
2. If you are experiencing permission-denied errors on Windows, run the command below in Powershell:
Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser